site/content/docs/main/backup-reference.md

```diff
@@ -60,3 +60,11 @@ velero backup create --from-schedule example-schedule
 ```
 
 This command will immediately trigger a new backup based on your template for `example-schedule`. This will not affect the backup schedule, and another backup will trigger at the scheduled time.
+
+## Enable Kubernetes API Pagination
+
+By default, Velero uses a single LIST API call for each resource type from the Kubernetes API when collecting items into a backup. This provides a fully consistent list. However, on clusters with a very large number of objects, the performance of this API call can be degraded and eventually be timed out by the Kubernetes apiserver.
```

Comment:
Let's rewrite this to reflect a default chunk size of 500.  There's no need to mention consistency since the code will handle an expired and fall back to the regular list so we should get a consistent list no matter what.